src/main/java/xml/XmlReader.java

```diff
@@ -94,18 +120,41 @@ public Document read(InputStream stream, String parentDir)
 	/**
 	 * Creates a DataFile from a file Element.
 	 * @return a new DataFile
+	 * @param elem The file element read from the xml file
+	 * @param parentDir The parent of the file
 	 */
 	public DataFile createDataFile(Element elem, String parentDir) {
```

Comment:
Als ik de methode goed begrijp doet hij 2 dingen, eerst het path bepalen. Vervolgens de data uit de file halen. Aangezien het een vrij lange methode is, zou ik de methode over twee methodes splitsen